src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -182,7 +185,7 @@ public function getNumberOfRequiredParameters(): int
      */
     public function getParameters(): array
     {
-        return $this->parameters;
+        return array_values($this->parameters);
```

Comment:
If these are indexed by name, is the order still guaranteed? 🤔

We should at least have a test covering a signature like `foo($c, $a, $b)` to ensure this doesn't change by accident

Reply:
Ou, you're right.

Reply:
Special test added.